llvm/test/CodeGen/AMDGPU/call-alias-register-usage3.ll

```diff
@@ -1,4 +1,5 @@
-; RUN: llc -O0 -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 < %s | FileCheck %s
+; RUN: sed 's/CODE_OBJECT_VERSION/400/g' %s | llc -O0 -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 | FileCheck -check-prefixes=CHECK,COV4 %s
+; RUN: sed 's/CODE_OBJECT_VERSION/500/g' %s | llc -O0 -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 | FileCheck -check-prefixes=CHECK,COV5 %s
```

Comment:
Don't really see the point of checking different code object versions for these register budget tests. The point is just to ensure aliases have sensible resource usage

Reply:
You are right. I have updated these tests to only check for cov5 in the latest commit.